SPECS/python-rpm-spec/python-rpm-spec.spec

```diff
@@ -0,0 +1,45 @@
+Name:           python-rpm-spec
+Version:        0.14.1
+Release:        1%{?dist}
+Summary:        A Python library for parsing RPM spec files.
+
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/bkircher/python-rpm-spec
+Source0:        https://github.com/bkircher/python-rpm-spec/archive/refs/tags/v0.14.1.tar.gz#/%{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+%description
+A Python library for parsing RPM spec files.
+
+%package -n       python3-rpm-spec
+Summary:          A Python library for parsing RPM spec files.
+
+%description -n python3-rpm-spec
+python-rpm-spec is a Python library for parsing RPM spec files.
+tl;dr If you want to quickly parse a spec file on the command line you might want to give rpmspec --parse a try.
+rpmspec --parse file.spec | awk '/Source/ {print $2}'
+If you write Python, have no /usr/bin/rpm around, or want to do something slightly more complicated, try using this Python library.
+RPMs are build from a package's sources along with a spec file. The spec file controls how the RPM is built. This library allows you to parse spec files and gives you simple access to various bits of information that is contained in the spec file.
+
+%prep
+%setup -q
+
+%install
+rm -rf %{buildroot}
+mkdir -p %{buildroot}/%{python3_sitelib}/
+cp -r pyrpm %{buildroot}/%{python3_sitelib}/
+
+
+%files -n python3-rpm-spec
+%license LICENSE
+%doc README.md
+%{python3_sitelib}/*
+
+
+%changelog
+* Tue Aug 15 2023 Andy Zaugg <azaugg@linkedin.com> - 0.14.1
+- Initial creation of RPM
```

Comment:
Sorry I meant to replace this line. You can get rid of 'Initial creation of RPM' line.
Further, as Pawel pointed out, please use the complete changelog from Fedora, and add Mariner specific import information.
Something like 
- Initial CBL-Mariner import from Fedora [give Fedora version] (license: MIT).
- License verified.

Reply:
Hey neha170, Do you want me to add the license of the project in its stead?  I generated the spec file with the help of rpmdev-newspec. I'll remove the line you suggested

Reply:
Some clarification: the `license: MIT` part in the changelog refers to the license of the **disto**, from which the spec was taken. The `License` tag at the top of the spec refers to the license of the **project** itself. Thus it's (almost) always MIT for Fedora specs, while the `License` tag differs from one spec to another.

Having said that, @AZaugg could you clarify how you arrived at creating this spec file? The `%fedora` macros suggested it was taken from Fedora, but your mention of `rpmdev-newspec` made me question that assumption.

Reply:
Hey @PawelWMS I used the template generated from `rpmdev-newspec` and packaged the contents of the upstream package. I didn't pull the spec file from another project.